### PR TITLE
MAP-1728 merging with hmpps-kotlin-template

### PR DESCRIPTION
**Pull request by michalnawrockidj**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -7,7 +7,7 @@ plugins {
   id("uk.gov.justice.hmpps.gradle-spring-boot") version "6.0.7"
   kotlin("plugin.spring") version "2.0.21"
   kotlin("plugin.jpa") version "2.0.21"
-  id("org.jetbrains.kotlinx.kover") version "0.9.0-RC"
+  id("org.jetbrains.kotlinx.kover") version "0.8.3"
```
Comment (Mjwillis) [inline build.gradle.kts:10]: This a lower?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+version: "3"
```
Comment (Mjwillis) [inline docker-compose.yml:1]: version not needed

**Review by Mjwillis — APPROVED** [review]
Some minor comments

File: build.gradle.kts (hunk 1 above)
Comment (michalnawrockidj) [reply build.gradle.kts:10]: Bit worry to use RC. Im more happy to stick with stable version

File: docker-compose.yml (hunk 2 above)
Comment (michalnawrockidj) [reply docker-compose.yml]: sorted

**Review by Mjwillis — APPROVED** [review]
LGTM

**Pull request merged**